game.js

```diff
@@ -0,0 +1,132 @@
+let compPoints = 0;
+let playerPoints = 0;
+let playerSelection = "";
+let computerSelection = "";
+let body = document.querySelector(".results-container");
+let resultsContainer = document.createElement('div');
+resultsContainer.className = "results";
+body.append(resultsContainer);
+let results = document.querySelector('.results');
+let scoreContainer = document.querySelector('.scorecard');
+let playerScore = document.createElement('div');
+playerScore.className = "item";
+scoreContainer.appendChild(playerScore);
+let compScore = document.createElement('div');
+compScore.className = "item";
+scoreContainer.appendChild(compScore);
```

Comment:
pls be mindful of where to use let & where to use const.

Kindly update

Reply:
okay mam. Done